### ISSUE TITLE
VERGE deamon doesn't get killed on programm close

### ISSUE THREAD

**Issue by marpme** [vergecurrency/vWallet#5]
**Describe the bug**
Verge deamon runs silently, while I'm closing the window. Need to manually kill it via the acitivity manager/task manager

**Expected behavior**
Should kill on close

**Comment by justinvforvendetta** [comment]
could send an rpc command for "stop"
ex: ./VERGEd stop

**Comment by marpme** [comment]
@justinvforvendetta well, that wouldn't work, when a user wants to shutdown the wallet too early. 

So imagine the verge daemon is still loading all the blocks in the background and you are going to kill the main process (vWallet) and then we would have to wait until the daemon finished loading to execute the stop command, that's not really applicable.

I also had a lot of tries to kill the verge daemon in harder ways:

Spawning verge wallet and saving child process reference
```js
  vergeProcess = childProcess.spawn(
    processPath,
    [`-rpcuser=${auth.user}`, `-rpcpassword=${auth.pass}`, '-printtoconsole'],
    // more options, with stdout, stdin etc.
  )
```

and when closing all vWallet windows or kill the process hard, I tried to kill the process as well:

```js
app.on('window-all-closed', () => {
  if (process.env.NODE_ENV !== 'dev') {
    vergeProcess.kill('SIGINT')
  }
  // quits the final app 
  app.quit()
})
```

I also did an while loop, where it checks if the wallet was still active
```js
while (!vergeProcess.killed) {
  vergeProcess.kill('SIGINT')
}
```
and it never reacted on it in any ways, maybe there's some code, which overwrites hard kills by other signals. Hope you have some insights, it's a quite show stopper at the moment :(
